occurrence-search/src/main/java/org/gbif/occurrence/search/es/OccurrenceEsField.java

```diff
@@ -110,6 +108,10 @@
   TAXON_ID("gbifClassification.taxonID", DwcTerm.taxonID),
   VERBATIM_SCIENTIFIC_NAME("gbifClassification.verbatimScientificName", GbifTerm.verbatimScientificName),
 
+  // GrSciColl
+  COLLECTION_KEY("grscicoll.collectionKey", GbifInternalTerm.collectionKey),
+  INSTITUTION_KEY("grscicoll.institutionKey", GbifInternalTerm.institutionKey),
```

Comment:
Since we're using "...key" terminology, I think it is probably ok to skip the "grscicoll." prefix